src/battle/anim/flying.c

```diff
@@ -23,6 +31,288 @@ void sub_80DB458(struct Sprite *sprite);
 void sub_80DB564(struct Sprite *sprite);
 void sub_80DB5E4(struct Sprite *sprite);
 
+void sub_80DA0DC(u8 taskId);
+
+void sub_80DA034(struct Sprite *sprite) 
+{
+    sub_8078764(sprite, 0);
+    sprite->pos1.y += 0x14;
```

Comment:
Use non-hexadecimal numbers for basic operations like this.  In general it's better to reserve hexadecimal for things like bitwise operations and sometimes Ids, such as the sprite palette tags.